include/cuco/detail/trie/bit_vector/bit_vector_ref.cuh

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <cuco/operator.hpp>
+
+namespace cuco {
+namespace experimental {
```

Comment:
This is declared in public namespace, but we want it under detail namespace.

Reply:
should all bitvector code go under coco::detail?